config/routes.rb

```diff
@@ -9,6 +9,7 @@
 
   get 'static_place', to: 'static#place'
   get '/autocomplete', to: 'google/autocomplete#index'
+  get '/map', to: 'static#map'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.